simnode/generator.py

```diff
@@ -67,9 +68,10 @@ def _interpolate_nodes(old_x_vals: List[float], old_y_vals: List[float], old_wid
             main_nodes = list(zip(new_x_vals, new_y_vals, z_vals, new_width_vals))
             main_road.nodes.extend(main_nodes)
             scenario.add_road(main_road)
-            # FIXME Recognize changing widths
+            # FIXME Recognize changing widths --- mostly done
```

Comment:
If you extend the comment please be a little more precise than "mostly done". So in one or two sentences: What works? What does not? 😉